app/src/main/java/org/android/go/sopt/ui/main/home/adapter/MultiViewHolderFactory.kt

```diff
@@ -0,0 +1,28 @@
+package org.android.go.sopt.ui.main.home.adapter
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import org.android.go.sopt.R
+
+class MultiViewHolderFactory {
+    @Suppress("UNCHECKED_CAST")
+    fun getViewHolder(parent: ViewGroup, viewType: MultiViewType): MultiViewHolder<MultiViewItem> {
+        return when (viewType) {
+            MultiViewType.TEXT ->
+                MultiViewHolder.TextViewHolder(viewBind(parent, R.layout.item_text))
+            MultiViewType.IMAGE ->
+                MultiViewHolder.ImageViewHolder(viewBind(parent, R.layout.item_image))
+        } as MultiViewHolder<MultiViewItem>
+    }
```

Comment:
```suggestion
import org.android.go.sopt.ui.main.home.adapter.MultiViewType.IMAGE
import org.android.go.sopt.ui.main.home.adapter.MultiViewType.TEXT

    fun getViewHolder(parent: ViewGroup, viewType: MultiViewType): MultiViewHolder<MultiViewItem> {
        return when (viewType) {
            TEXT ->
                MultiViewHolder.TextViewHolder(viewBind(parent, R.layout.item_text))
            IMAGE ->
                MultiViewHolder.ImageViewHolder(viewBind(parent, R.layout.item_image))
        } as MultiViewHolder<MultiViewItem>
    }
```

사람마다 코드 스타일이 다르지만 저는 상수값 임포트를 통해 코드를 간략화하는 편입니다!